example/spec/parallel/ingestGranule/IngestGranuleWithDuplicateHandlingErrorSpec.js

```diff
@@ -0,0 +1,214 @@
+'use strict';
+
+const get = require('lodash/get');
+const pAll = require('p-all');
+const pick = require('lodash/pick');
+const { randomId } = require('@cumulus/common/test-utils');
+
+const { createCollection } = require('@cumulus/integration-tests/Collections');
+const {
+  findExecutionArn, getExecutionWithStatus
+} = require('@cumulus/integration-tests/Executions');
+const { getGranuleWithStatus } = require('@cumulus/integration-tests/Granules');
+const { createProvider } = require('@cumulus/integration-tests/Providers');
+const { createOneTimeRule } = require('@cumulus/integration-tests/Rules');
+
+const { deleteCollection } = require('@cumulus/api-client/collections');
+const { deleteGranule } = require('@cumulus/api-client/granules');
+const { deleteProvider } = require('@cumulus/api-client/providers');
+const { deleteRule } = require('@cumulus/api-client/rules');
+
+const { deleteS3Object, s3PutObject } = require('@cumulus/aws-client/S3');
+
+const { loadConfig } = require('../../helpers/testUtils');
```

Comment:
I'm assuming there were issues with the various locations within this that testUtils was refactored away?